src/org/labkey/test/components/ui/files/FileAttachmentContainer.java

```diff
@@ -0,0 +1,249 @@
+package org.labkey.test.components.ui.files;
```

Comment:
Given the choice to make between implementing the functionality of a component in the Biologics AssayUpload page and writing a wrapper for a component we don't really have a (potentially reusable) wrapper for, I chose to make this after discussing with Cory which components are in play on the dev side